src/internal.c

```diff
@@ -16652,29 +16657,12 @@ int SetCipherList(WOLFSSL_CTX* ctx, Suites* suites, const char* list)
                     return 0; /* suites buffer not large enough, error out */
                 }
 
-                suites->suites[idx++] =
-            #ifdef WOLFSSL_TLS13
```

Comment:
Why are these no longer needed?

Reply:
The cipher_names array has since had the first byte of the cipher suite ID added to it. Parsing by string was confusing the IANA version with the TLS1.3 cipher suite additions and is an extra step that can be removed by the additions that were done previously to the cipher_names array.

Reply:
Ok, thanks.